resources/assets/js/app/views/mainLayout.js

```diff
@@ -13,9 +14,33 @@ var mainLayoutView = Marionette.LayoutView.extend({
         breadCrumbs: '#breadcrumbs',
         modalWindow: '#modal-container'
     },
+    onRender: function () {
+        if(config.debug === false){
```

Comment:
Probably there is can be more suitable condition. Please take a look environment settings.